js/ui/popup/ui.popup.js

```diff
@@ -419,7 +425,10 @@ const Popup = Overlay.inherit({
     },
 
     _renderVisibilityAnimate: function(visible) {
-        this._observeContentResize(visible);
+        // NOTE: T1130045
+        if(!visible) {
```

Comment:
Couldn't we unsubscribe in another place to get rid of "if" condition? For example, `_hide` method?

Reply:
Move unsubscription to `_hide` method [here](https://github.com/DevExpress/DevExtreme/blob/85588cf6448360a32ce9f12134dd6d362059ef84/js/ui/popup/ui.popup.js#L431).